### ISSUE TITLE
False-positive NOT NULL check on AddField with default

### ISSUE THREAD

**Issue by Evgenmater** [3YOURMIND/django-migration-linter#326]
Running `python manage.py lintmigrations` reports a `NOT NULL` constraint issue for `AddField` even when a default is provided.

Example:

```python
    migrations.AddField(
        model_name="example",
        name="flag",
        field=models.BooleanField(default=False),
    )
```

Command output:

`    NOT NULL constraint on columns`

**Comment by mrmikardo** [comment]
I have this issue too, and I get around it in a kludgy way: by using `linter.IgnoreMigration()` in any migration which adds such a column.
